specification/postgresql/resource-manager/readme.md

```diff
@@ -57,7 +57,8 @@ These settings apply only when `--tag=package-2017-12-01` is specified on the co
 
 ``` yaml $(tag) == 'package-2017-12-01' 
 input-file: 
-- Microsoft.DBforPostgreSQL/stable/2017-12-01/postgresql.json 
+- Microsoft.DBforPostgreSQL/stable/2017-12-01/postgresql.json
+- Microsoft.DBforPostgreSQL/stable/2017-12-01/securityAlertPolicies.json

```

Comment:
We only maintain one file for APIs. Please merge your policies API into PostgreSQL.json file.

Reply:
done